testing/run-scripts/run_cloud.sh

```diff
@@ -64,7 +65,17 @@ then
     fi
     BANNER=`docker exec uproxy-sshd cat /banner`
   else
-    BANNER=`hostname`
+    # DigitalOcean's metadata API can tell us the region in which
+    # the machine is located:
+    #   https://developers.digitalocean.com/documentation/metadata/#metadata-api-endpoints
+    BANNER=`curl -s -m 2 http://169.254.169.254/metadata/v1/region`
+    if [ ! -z "$BANNER" ]
+    then
+      BANNER=`echo "$BANNER"|sed 's/ams./Netherlands/;s/sgp./Singapore/;s/fra./Germany/;s/tor./Canada/;s/nyc./New York, USA/;s/sfo./San Francisco, USA/;s/lon./UK/'`
+      BANNER="$BANNER (DigitalOcean)"
```

Comment:
The script could also prompt the user to supply a unique name as the first thing it does, offer a sensible default, and use the default after 10 seconds of inactivity. (maybe too much, again)